### PR TITLE
feature/live-now-marquee

### PR DESCRIPTION
**Pull request by PaulieScanlon**

### CODE REVIEW

File: gatsby-browser.js [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
 export const wrapPageElement = ({ element }) => {
   return <Seo>{element}</Seo>
 }
+
+// wrapRootElement persits across routes and doesn't unmount,
+// we might want this to be in wrapPageElement above
+// maybe a good point to discuss the difference
+export const wrapRootElement = ({ element }) => {
+  return (
+    <Fragment>
```
Comment (raae) [inline gatsby-browser.js:15]: Any reason not to use `<></>`?

**Comment by raae** [comment]
**suggestion:** Should we also add it to gatsby-ssr so it does not shift down, could be solved with opacity 0.

**suggestion:** When not live should it be changed to "Next show <Date>"

File: src/components/live-now-marquee.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <span aria-label="Red Circle" role="img">
+          🔴
+        </span>
+
+        <marquee
+          style={{
+            backgroundColor: "#ff0000",
+          }}
+        >
+          Live Now | https://www.youtube.com/raaecodes
```
Comment (raae) [inline src/components/live-now-marquee.js]: Is an `a` allowed inside `marquee`?

File: gatsby-browser.js (hunk 1 above)
Comment (PaulieScanlon) [reply gatsby-browser.js:15]: mainly preference. But... 

- `<Fragment />` encourages de-structured imports
-  it's easier to read
- it's easier to find and replace if you later wanna replace it with a DOM element or component IMO

File: src/components/live-now-marquee.js (hunk 2 above)
Comment (PaulieScanlon) [reply src/components/live-now-marquee.js]: not sure but if the whole thing is wapped in an `a` it'll still act as a link when `prefersReducedMotion` conditionally renders the `<marquee />`

**Comment by PaulieScanlon** [comment]
@raae i like both of your suggestions. Regarding `gatsby-ssr` - yeah we could and maybe we will when we build the real thing for now it's not essential for it to work or for it to work when we implement Functions. The layout shift can probably be resolved when we add styling as this may well end up being position fixed so it's always there even when you scroll.

Totally up to you how this looks and what it says, we can tinker with it on stream if you like?

**Pull request closed without merging**